src/neo-vm/ExecutionEngine.cs

```diff
@@ -289,7 +305,32 @@ private bool ExecuteInstruction()
                     }
                 case OpCode.THROW:
                     {
-                        return false;
+                        if (!TryPop(out StackItem error)) return false;
+                        return ExecuteThrow(error);
```

Comment:
Is this the only place that throws the `CatcheableException`?

Reply:
Yes

Reply:
Then I think it should return `true`.

Reply:
Agree, `PostExecuteInstruction` should be executed.